metadata-ingestion/tests/unit/sql_parsing/test_sqlglot_utils.py

```diff
@@ -8,6 +8,7 @@
 from datahub.sql_parsing.sql_parsing_common import QueryType
 from datahub.sql_parsing.sqlglot_lineage import _UPDATE_ARGS_NOT_SUPPORTED_BY_SELECT
 from datahub.sql_parsing.sqlglot_utils import (
+    _TABLE_NAME_NORMALIZATION_RULES,
```

Comment:
we should remove the `_` prefix if this is a public thing that other sources depend on